### DISCUSSION TITLE
Bug? - Cannot deselect "Included Chains"

### DISCUSSION THREAD

**Discussion by WolfPG** [pwillworth/dfkreport#30] [General]
Hi,

I'm having an issue which i'm not sure if it is a bug?

Basically, I select that I only want to generate a report for say DFK Chain. I generate the report, and then after the report is generated, in the left bar it has other chains highlighted as included. I click on them to deselect them but nothing happens. I can see in the report's "Transactions" that it has a lot more transactions than I know I have on the chain I originally selected prior to generating the report.

When I then export the Koinly csv, I can see that it has included transactions from other chains too.

This is most evident with Klaytn network as I have only 26 transactions, yet the csv and report contains 11,000+ transactions, which could only have come from Harmony and DFK Chain transactions.

Please fix this if it is a bug, or advise me as to how I resolve this if possible.

Many thanks in advance!

**Comment by pwillworth** [comment]
Hi, thanks for the report.  Sounds like it's probably a bug, I'll take a look soon

**Reply by WolfPG** [reply]
Awesome! Thanks :D

**Comment by pwillworth** [comment]
This bug has been found and squashed.  You should get correct set on your transactions now.

**Reply by WolfPG** [reply]
Amazing! I can confirm it is now working. Thanks for resolving that so quickly, I really appreciate it. 👍
